flixel/addons/editors/ogmo/FlxOgmo3Loader.hx

```diff
@@ -0,0 +1,514 @@
+package flixel.addons.editors.ogmo;
+
+import flixel.math.FlxPoint;
+import flixel.group.FlxGroup;
+import flixel.FlxG;
+import flixel.tile.FlxTilemap;
+import flixel.math.FlxRect;
+
+using flixel.addons.editors.ogmo.FlxOgmo3Loader;
+using openfl.Assets;
+using haxe.Json;
+using Math;
+using Reflect;
+
+class FlxOgmo3Loader
+{
+	var project:ProjectData;
+	var level:LevelData;
+
+	/**
+	 * Creates a new instance of `FlxOgmoLoader` and prepares the XML level data to be loaded.
+	 * This object can either be contained or ovewritten.
+	 *
+	 * IMPORTANT:
+	 *  * Tile layers must have the Export Mode set to `"CSV"`.
+	 *  * First tile in spritesheet must be blank or debug. It will never get drawn so don't place them in Ogmo!
+	 *    (This is needed to support many other editors that use index `0` as empty)
+	 *
+	 * @param	LevelData	A String or Class representing the location of xml level data.
+	 */
+	public function new(ProjectData:String, LevelData:String)
+	{
+		project = ProjectData.getText().parseProjectJSON();
```

Comment:
Not a big fan of using static extensions for `openfl.Assets`...

Reply:
👍